Assets/Shopify/UIToolkit/Themes/Generic/GenericSingleProductTheme.cs

```diff
@@ -18,6 +18,9 @@ public class GenericSingleProductTheme : MonoBehaviour, ISingleProductTheme {
         public GameObject VariantDropdownMenu;
         public GameObject VariantPicker;
         public GameObject VariantPickerTitle;
+        public GameObject ActiveImage;
+        public GameObject ImageContainer;
+        public GameObject ImageHolderTemplate;
```

Comment:
Why are these all game object? isnt it better to have them explicitly reffered to as an `Image` so we dont have to rely on a null-unsafe `GetComponent` call?

Reply:
I only thought you could drag GameObjects are script references but boy was I wrong! This is cool I didn't know you can do this.